packages/utils/src/json-schema.ts

```diff
@@ -1,24 +1,2 @@
-// Note - @types/json-schema@7.0.4 added some function declarations to the type package
-// If we do export *, then it will also export these function declarations.
-// This will cause typescript to not scrub the require from the build, breaking anyone who doesn't have it as a dependency
```

Comment:
yeah because in the olden days we did `export * from 'json-schema'` and we'd export the types for functions that don't actually exist.
But now a days we can do `export type *` and it'll not include the runtime bits!

Though #6963 would just be better than making this change at all TBH